edalize/symbiflow.py

```diff
@@ -52,31 +59,138 @@ def get_doc(cls, api_ver):
                     {
                         "name": "pnr",
                         "type": "String",
-                        "desc": 'Place and Route tool. Currently only "vpr" is supported',
+                        "desc": 'Place and Route tool. Currently only "vpr"/"vtr" and "nextpnr" are supported',
                     },
                     {
-                        "name": "vpr_options",
+                        "name": "pnr_options",
```

Comment:
I think we should keep vpr_options and add a separate nextpnr_options instead for two reasons
1. It will break builds for users already using vpr_options
2. It's easier for users if they can specify correct options for both vpr and nextpnr and then just change the `pnr` option to switch between the different tools

Reply:
You're right - reverted to `vpr_options` / `nextpnr_options`